azurelinuxagent/common/osutil/openbsd.py

```diff
@@ -35,12 +35,14 @@
     r'^\s*[A-F0-9]{8}(?:\-[A-F0-9]{4}){3}\-[A-F0-9]{12}\s*$',
     re.IGNORECASE)
 
+
 class OpenBSDOSUtil(DefaultOSUtil):
 
     def __init__(self):
         super(OpenBSDOSUtil, self).__init__()
         self.jit_enabled = True
         self._scsi_disks_timeout_set = False
+        self.service_name = self.get_service_name()
```

Comment:
You don't need to define self.service_name here once it is defined in DefaultOSUtil.

Reply:
Good point, thanks.

Reply:
Actually, I take that back, the `self.service_name` doesn't resolve if it's not defined in the current class constructor.